netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -208,14 +208,24 @@ const MapSettings = ({
           <MenuItem onClick={() => onSensorChange(!showSensors)}>
             <Checkbox checked={showSensors} color="default" /> Sensors
           </MenuItem>
-          <MenuItem disabled onClick={() => onHeatmapChange(!showHeatmap)}>
+          <MenuItem onClick={() => onHeatmapChange(!showHeatmap)}>
             <Checkbox checked={showHeatmap} color="default" /> Heatmap
           </MenuItem>
           <Divider />
-          <MenuItem onClick={() => onCalibratedChange(!showCalibratedValues)}>
-            <Checkbox checked={showCalibratedValues} color="default" />{" "}
-            Calibrated values
-          </MenuItem>
+          {showSensors ? (
+            <MenuItem onClick={() => onCalibratedChange(!showCalibratedValues)}>
+              <Checkbox checked={showCalibratedValues} color="default" />{" "}
+              Calibrated values
+            </MenuItem>
+          ) : (
```

Comment:
Calibrated values should not appear by default and should only appear when sensors are shown.